indexer/httpd/src/routes/index.rs

```diff
@@ -1,6 +1,45 @@
-use actix_web::{Responder, get};
+use {
+    crate::context::Context,
+    actix_web::{Error, HttpResponse, Responder, error::ErrorInternalServerError, get, web},
+    async_graphql::futures_util::TryFutureExt,
+    grug_app::GIT_COMMIT,
+    indexer_sql::entity,
+    sea_orm::{EntityTrait, Order, QueryOrder},
+};
 
 #[get("/")]
 pub async fn index() -> impl Responder {
     "OK"
 }
+
+#[derive(serde::Serialize, Default)]
+struct HealthResponse<'a> {
+    block_height: u64,
+    indexed_block_height: Option<u64>,
+    git_commit: &'a str,
+}
```

Comment:
Consider renaming this to something else, because we already have a `HealthResponse` somewhere else (the health of a margin account's loan position)